pkg/monitor/monitor/util.go

```diff
@@ -118,10 +118,23 @@ func getMonitorConfigMap(tc *v1alpha1.TidbCluster, dc *v1alpha1.DMCluster, monit
 
 	var releaseClusterInfos []ClusterRegexInfo
 	for _, cluster := range monitor.Spec.Clusters {
-		releaseClusterInfos = append(releaseClusterInfos, ClusterRegexInfo{
+		clusterRegex := ClusterRegexInfo{
 			Name:      cluster.Name,
 			Namespace: cluster.Namespace,
-		})
+		}
+		//  Old logical , firstTc enable tls and use default tls path
```

Comment:
We can check if the TLS enabled for each TC, why do we need to use the original logic here?